crates/iota/src/name_commands.rs

```diff
@@ -90,7 +90,9 @@ pub enum NameCommand {
         /// records will be returned.
         key: Option<String>,
     },
-    /// List the names owned by the given address, or the active address
+    /// List the names and subnames owned by the given address, or the
+    /// active address. Note that leaf subnames are not listed by the `list`
+    /// command.
```

Comment:
```suggestion
    /// List the names and subnames owned by the given address, or the
    /// active address. Note that leaf subnames are not listed.
```
I think it's a bit weird that the command mentions itself